apps/src/organized_segmentation_demo.cpp

```diff
@@ -276,7 +281,7 @@ OrganizedSegmentationDemo::cloud_cb (const CloudConstPtr& cloud)
   ne.setInputCloud (cloud);
   ne.compute (*normal_cloud);
   float* distance_map = ne.getDistanceMap ();
-  pcl::EdgeAwarePlaneComparator<PointT, pcl::Normal>::Ptr eapc = boost::dynamic_pointer_cast<pcl::EdgeAwarePlaneComparator<PointT, pcl::Normal> >(edge_aware_comparator_);
+  pcl::EdgeAwarePlaneComparator<PointT, pcl::Normal>::Ptr eapc = std::dynamic_pointer_cast<pcl::EdgeAwarePlaneComparator<PointT, pcl::Normal> >(edge_aware_comparator_);
```

Comment:
Should we have a `pcl::dynamic_pointer_cast` or just unqualify this one to force ADL?

Reply:
It would fit right in in memory.h, wouldn't it?

Reply:
Ah, that's for 1.11.0, so I guess it would be something to put inside make_shared.h and immediately move to memory.h, in case, right?

Reply:
Yes, that would be ideal. I do hope the other PR gets merged soon 😅

Reply:
I probably would prefer to go with unqualified, but I'll let you guys decide.

Reply:
In this situation I would rather go unqualified and force ADL. The less extra symbols we need to define are less symbols we need to deprecate later.